.travis.yml

```diff
@@ -3,10 +3,12 @@ language: c
 compiler:
   - i686-w64-mingw32-gcc
   - x86_64-w64-mingw32-gcc
+  - gcc
```

Comment:
Please specify this compiler as `x86_64-linux-gnu-gcc` (64bit) or `i686-linux-gnu-gcc` (32bit), or if you like, both. So like for mingw we would know if it is 32 or 64bit.